.github/workflows/fledge.yaml

```diff
@@ -0,0 +1,26 @@
+name: fledge
+
+on:
+  # for manual triggers
+  workflow_dispatch:
+  # daily run
+  schedule:
+  - cron: "30 14 * * *"
+
+jobs:
+  pkgcheck:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: r-lib/actions/setup-r@v2
+        with:
+          use-public-rspm: true
+
+      - name: Install fledge
+        run: Rscript -e 'install.packages("fledge")'
+
+      - name: Bump version
+        run: Rscript -e 'fledge::bump_version(which = "dev")'
```

Comment:
Will this work? Should we:

- install from `"."`
- add a flag to `bump_version()` ?

Reply:
> install from `"."`

oh right fledge repo itself is a special case.

> add a flag to `bump_version()` ?

what do you mean?

Reply:
`fledge::bump_version(no_change_behavior = "noop")`